tests/kafkatest/services/kafka/kafka.py

```diff
@@ -887,6 +891,11 @@ def start_node(self, node, timeout_sec=60, **kwargs):
                     else:
                         cmd += " --standalone"
                         self.standalone_controller_bootstrapped = True
+            if self.use_transactions_v2:
+                cmd += " --feature transaction.version=2"
```

Comment:
We missed setting `transaction.version` for isolated kraft, resulting in the cluster using v0. Since this nullifies the suit `use_transactions_v2=True`, we will submit a patch later.

Reply:
I observed this issue while running tests/kafkatest/tests/core/transactions_test.py

The settings "isolated_kraft" and "use_transactions_v2=true" failed to enable tv2 on the cluster

Reply:
> Do we know how this maps to the kafka.py file? Ie, why this line of code doesn't apply for isolated kraft?

see #21164